docs/user/exercises/programming-exercise-setup.inc

```diff
@@ -75,7 +75,7 @@ Instructors can still use those templates to generate programming exercises and
 
 We encourage instructors to contribute improvements to the existing ``templates`` or to provide new templates. Please contact Stephan Krusche and/or create Pull Requests in the Github repository.
 
-
+.. _exercise-creation:
```

Comment:
I think this can be reverted since your tests regarding relative links failed, can't it?